functions.md

```diff
@@ -265,6 +265,23 @@ mod(10, 2.1)  => error
 mod(10, "2")  => error
 ```
 
+### _hashint(object) -> int_
+
+Returns a hash value of the given object. It differs from `sha256-hex` in that
+it gives a numeric integral hash value instead of a string. There is no guarantee that the same
+input will produce the same output in different environments (JVM versions, etc.)
+
+Examples:
+
+```
+hash-int("test") => 1083642606
+hash-int("") => 1088
+hash-int({}) => 119320
+hash-int([]) => 88536
+hash-int(1) => 49
+hash-int(null) => null
+```
```

Comment:
This one is much less extensive than the examples in the other PR. I think we should keep the examples for that one, and with the same properties.

Reply:
copied the examples from the other PR, I didn't bother making sure that the actual return values match the current implementation since it's not guaranteed to have the same values across different JVM implementations and versions.